### ISSUE TITLE
Bootstrapped data and relationships with `peekAll`

### ISSUE THREAD

**Issue by snewcomer** [samselikoff/ember-cli-mirage#580]
If I have bootstrapped data and a model called Foo that has a belongsTo relationship to one of those bootstrapped models (let's call it `Status`).  
```
//acceptance test
    server.loadFixtures('statuses');
    const foo = server.create('foo', {statusId: server.db.statuses[0].id});
```
In the route model hook, I call `store.peekAll('status')` because the `status` is already loaded in the store..  However, this does not setup the relationship `foo.status.id`.  When I call store.findAll('status'), the belongsTo relationship is setup between Foo and Status.

Is it possible for mirage to setup relationships with bootstrapped data where there isn't an http call in the route?  Or is there a more manual setup process that I have to go through b/w `Foo` and `Status`.

**Comment by samselikoff** [comment]
How will the data be loaded in your actual production environment?

**Comment by snewcomer** [comment]
Backend provides it it on boot.  Closing this issue as I seemed to have fixed this in the test environment.  Not sure what fixed it though.  In the learning phase...sry about that.

One thing that would be help newbies get off the ground faster is documentation on helpful debugging techniques. Some of the trouble I have run into is with relationships and understanding exactly what is required. For example having a relationship with `assignee: belongsTo('person')` and using `assigneeId` when I needed to use `personId` as the explicit link.  Solved it through just trying new things out.  Otherwise, no complaints here!

**Comment by samselikoff** [comment]
@snewcomer thank you for your feedback. Actually, it is `assigneeId`, there was a bug that has been fixed in 2400b33a1e6bd9371436875b684bc6ae943f6b28 - sorry about that! We'll be cutting a new release soon.

Also, I'd love to have a tab in the Ember Inspector to show precisely the kind of information you're talking about: how your schema is set up, what the foreign keys are etc.
